WordPress/Classes/ViewRelated/Stats/Period Stats/SiteStatsPeriodViewModel.swift

```diff
@@ -104,4 +106,29 @@ private extension SiteStatsPeriodViewModel {
         return dataRows
     }
 
+    func publishedTableRows() -> [ImmuTableRow] {
+        var tableRows = [ImmuTableRow]()
+        tableRows.append(CellHeaderRow(title: PeriodHeaders.published))
+        tableRows.append(TopTotalsNoSubtitlesPeriodStatsRow(dataRows: publishedDataRows(),
+                                                            siteStatsPeriodDelegate: periodDelegate))
+
+        return tableRows
+    }
+
+    func publishedDataRows() -> [StatsTotalRowData] {
```

Comment:
Same thing as in #10895 — this could be a simple `map` :)

Reply:
And updated. Thanks!!